examples/js/renderers/CSS3DRenderer.js

```diff
@@ -13,11 +13,15 @@ THREE.CSS3DObject = function ( element ) {
 
 	this.addEventListener( 'removed', function () {
 
-		if ( this.element.parentNode !== null ) {
+		this.traverse( function( object ) {
 
-			this.element.parentNode.removeChild( this.element );
+			if ( object.element !== null && object.element.parentNode !== null ) {
```

Comment:
`object.element` could be `undefined` or other value. How about this?
```suggestion
			if ( object.element instanceof Element && object.element.parentNode !== null ) {
```